docs/conf/australia/2024/news/tickets-closing-get-ready.rst

```diff
@@ -0,0 +1,54 @@
+:template: {{year}}/generic.html
+
+.. post:: October 22, 2024
+   :tags: {{shortcode}}-{{year}}, tickets, visiting, sponsors
+
+Get ready for Write the Docs Australia 2024
+============================================
+
+We're just over 4 weeks out from the conference, and we hope everyone is getting excited!
+
+In this update we share a few updates as the conference approaches.
+
+Ticket sales closing soon
+--------------------------
+
+We will be closing ticket sales **Saturday, 16th November**, so if you've been on the fence, now is the time to buy one!
+
+If you were hoping to come to this years event,
+we recommend that you `get your tickets soon <https://www.writethedocs.org/conf/australia/2024/tickets/>`_.
+
+Things to prepare for
+---------------------
+
+* **Read the Visiting Melbourne information**. Our `Visiting {{city}} <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/visiting/>`__ page contains all the information you need for finding food, transit, and other things to do while you're in town.
+* **Review the schedule.** The `Schedule page <https://www.writethedocs.org/conf/australia/{{year}}/schedule/>`_ has the latest and most complete information about the presentations. This is where we will also update any changes/cancellations if they occur.
+* **Review the Code of Conduct.** We have a wonderful community full of diverse and amazing people. We expect everyone to behave in a respectful manner at all conference events and online channels. Our `Code of Conduct <https://www.writethedocs.org/code-of-conduct/>`_ has all the information.
+
+Get involved at the conference
+------------------------------
+
+-  **Attend the Brick Docs workshop.** This year, similar to our last few conferences, we will have a workshop on Day 1 after the lightning talks.
```

Comment:
No link for what it is?

Reply:
The workshop page is not up yet - could be a couple more weeks before we get it up :(